process/block/shardblock.go

```diff
@@ -257,6 +251,35 @@ func (sp *shardProcessor) ProcessBlock(
 	return nil
 }
 
+func (sp *shardProcessor) getMetricsFromBlockBody(body block.Body) {
```

Comment:
do this in metrics.go - add these code there.

Reply:
Done. Methods that set metrics was moved in a new file.